### PR TITLE
fix(sec): upgrade org.apache.commons:commons-dbcp2 to 2.9.0

### PR DESCRIPTION
**Pull request by sd8089730**
### What happened？
There are 1 security vulnerabilities found in org.apache.commons:commons-dbcp2 2.8.0
- [MPS-2022-12024](https://www.oscs1024.com/hd/MPS-2022-12024)

### What did I do？
Upgrade org.apache.commons:commons-dbcp2 from 2.8.0 to 2.9.0 for vulnerability fix

### What did you expect to happen？
Ideally, no insecure libs should be used.

### The specification of the pull request
[PR Specification](https://www.oscs1024.com/docs/pr-specification/) from OSCS

### CODE REVIEW

**Comment by jonatan-ivanov** [comment]
Could you please provide a link to this that all of the maintainers can understand (english is fine)?

File: pom.xml [hunk 1]
```diff
@@ -13,11 +13,7 @@
   ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
-  -->
-
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (jonatan-ivanov) [inline pom.xml:18]: Please do not reformat this.

File: pom.xml [hunk 2]
```diff
@@ -510,4 +506,4 @@
 		</profile>
 	</profiles>
 
-</project>
+</project>
```
Comment (jonatan-ivanov) [inline pom.xml:509]: Please do not reformat this.

File: pom.xml [hunk 3]
```diff
@@ -75,7 +71,7 @@
 		<p6spy.version>3.9.1</p6spy.version>
 		<datasource-proxy.version>1.7</datasource-proxy.version>
 		<tomcat-jdbc.version>10.0.6</tomcat-jdbc.version>
-		<commons-dbcp2.version>2.8.0</commons-dbcp2.version>
+		<commons-dbcp2.version>2.9.0</commons-dbcp2.version>
```
Comment (jonatan-ivanov) [inline pom.xml:74]: Is there a patch version (2.8.x) of dbcp where this issue is fixed?

**Review by jonatan-ivanov — CHANGES_REQUESTED** [review]

**Comment by marcingrzejszczak** [comment]
Ping @sd8089730

**Pull request closed without merging**